packages/manager/modules/hub/src/components/user-panel/shortcuts/translations/Messages_fr_FR.json

```
@@ -0,0 +1,9 @@
+{
+  "hub_user_panel_shortcuts_title": "Raccourcis",
+  "hub_user_panel_shortcuts_link_services": "Produits et services",
+  "hub_user_panel_shortcuts_link_bills": "Factures",
+  "hub_user_panel_shortcuts_link_assistance": "Niveau de support",
+  "hub_user_panel_shortcuts_link_products": "Catalogue de produits",
+  "hub_user_panel_shortcuts_link_emails": "Emails de service",
```

Unless it is made by content team
```suggestion
  "hub_user_panel_shortcuts_link_emails": "E-mails de service",
```